ibm/service/vpc/data_source_ibm_is_instance_profile.go

```diff
@@ -1289,6 +1314,26 @@ func dataSourceInstanceProfileVcpuArchitectureToMap(vcpuArchitectureItem vpcv1.I
 	return vcpuArchitectureMap
 }
 
+func dataSourceInstanceProfileFlattenClusterNetworkAttachmentIsolationPolicy(result vpcv1.InstanceProfileClusterNetworkAttachmentIsolationPolicy) (finalList []map[string]interface{}) {
+	finalList = []map[string]interface{}{}
+	finalMap := dataSourceInstanceProfileClusterNetworkAttachmentIsolationPolicyToMap(result)
+	finalList = append(finalList, finalMap)
+	return finalList
+}
+
+func dataSourceInstanceProfileClusterNetworkAttachmentIsolationPolicyToMap(vcpuArchitectureItem vpcv1.InstanceProfileClusterNetworkAttachmentIsolationPolicy) (vcpuArchitectureMap map[string]interface{}) {
+	vcpuArchitectureMap = map[string]interface{}{}
+
+	if vcpuArchitectureItem.Type != nil {
```

Comment:
change naming to cluster_network_attachment_isolation_policy and not vcpu architecture